2017-04-28 00:19:34.0643|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-28 00:19:34.0643|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-28 00:19:34.0984|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-28 00:19:34.0984|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-28 00:19:34.0984|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-28 00:19:34.1144|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-28 00:23:09.0532|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-28 00:23:34.8728|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|Exception during request processing.
System.AggregateException: One or more errors occurred. ---> System.Threading.Tasks.TaskCanceledException: A task was canceled.
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at Microsoft.Owin.StaticFiles.SendFileMiddleware.SendFileWrapper.<>c__DisplayClass2.<SendAsync>b__0(Task resultTask)
   at System.Threading.Tasks.ContinuationTaskFromTask.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.Owin.Host.HttpListener.OwinHttpListener.<ProcessRequestAsync>d__5.MoveNext()
---> (Inner Exception #0) System.Threading.Tasks.TaskCanceledException: A task was canceled.<---

2017-04-28 00:26:39.0702|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-28 00:26:39.0762|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-28 00:26:39.0762|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-28 00:26:39.1122|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-28 00:26:39.0762|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-28 00:26:39.0762|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-28 00:30:24.0874|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-28 00:34:39.1174|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-28 00:34:39.1244|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-28 00:34:39.1244|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-28 00:34:39.1244|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-28 00:35:14.1239|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-28 00:35:29.1264|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-28 00:42:24.1967|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-28 00:42:24.2023|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-28 00:44:34.2195|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-28 00:44:44.2253|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-28 11:31:06.5617|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-28 11:31:16.5574|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-28 12:51:53.9262|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-28 12:51:53.9262|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-28 12:51:53.9512|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-28 12:51:53.9512|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-28 12:54:06.9340|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-28 12:54:11.9365|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-28 14:49:54.7872|ERROR|Cafe2u.Application.WebApi.Helpers.NlogExceptionLogger|Request: http://localhost:12345/api/Groups/GetItems?filter[from]=0&filter[count]=30&filter[sortBy]=code&filter[IsAscending]=true&filter[categoryId]=1&filter[groupId]=1&_=1493380132243
Unhandled exception: System.ArgumentException: The Include path expression must refer to a navigation property defined on the type. Use dotted paths for reference navigation properties and the Select operator for collection navigation properties.
Parameter name: path
   at System.Data.Entity.QueryableExtensions.Include[T,TProperty](IQueryable`1 source, Expression`1 path)
   at Cafe2u.Application.DAL.Repositories.ItemsRepository.GetQueryableByFilter(ProductsFilter filter) in C:\work\DIPLOMA\Cafe2u\Cafe2u.Application.DAL\Repositories\ItemsRepository.cs:line 27
   at Cafe2u.Application.BLL.Helpers.EntityQueryFilterHelper`2.FilterData(TFilter filter) in C:\work\DIPLOMA\Cafe2u\Cafe2u.Application.BLL\Helpers\EntityQueryFilterHelper.cs:line 26
   at Cafe2u.Application.BLL.Services.ProductsService.GetItems(ProductsFilter filter) in C:\work\DIPLOMA\Cafe2u\Cafe2u.Application.BLL\Services\ProductsService.cs:line 51
   at Cafe2u.Application.WebApi.Contollers.ProductsController.GetItems(ProductsFilter filter) in C:\work\DIPLOMA\Cafe2u\Cafe2u.Application.WebApi\Contollers\ProductsController.cs:line 41
   at lambda_method(Closure , Object , Object[] )
   at System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.<>c__DisplayClass10.<GetExecutor>b__9(Object instance, Object[] methodParameters)
   at System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.Execute(Object instance, Object[] arguments)
   at System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ExecuteAsync(HttpControllerContext controllerContext, IDictionary`2 arguments, CancellationToken cancellationToken)
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__0.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__2.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Web.Http.Dispatcher.HttpControllerDispatcher.<SendAsync>d__1.MoveNext()
2017-04-28 14:49:54.8083|ERROR|Cafe2u.Application.WebApi.Helpers.NlogExceptionLogger|Request: http://localhost:12345/api/Groups/GetItems?filter[from]=0&filter[count]=30&filter[sortBy]=code&filter[IsAscending]=true&filter[categoryId]=1&filter[groupId]=1&_=1493380132243
Unhandled exception: System.ObjectDisposedException: Cannot access a disposed object.
Object name: 'System.Net.HttpListenerRequest'.
   at System.Net.HttpListenerRequest.CheckDisposed()
   at System.Net.HttpListenerRequest.get_LocalEndPoint()
   at System.Net.HttpListenerRequest.get_IsLocal()
   at Microsoft.Owin.Host.HttpListener.RequestProcessing.OwinHttpListenerContext.GetServerIsLocal()
   at Microsoft.Owin.Host.HttpListener.RequestProcessing.CallEnvironment.get_ServerIsLocal()
   at Microsoft.Owin.Host.HttpListener.RequestProcessing.CallEnvironment.PropertiesTryGetValue(String key, Object& value)
   at Microsoft.Owin.Host.HttpListener.RequestProcessing.CallEnvironment.TryGetValue(String key, Object& value)
   at Microsoft.Owin.OwinContext.Get[T](String key)
   at System.Web.Http.Owin.OwinHttpRequestContext.get_IsLocal()
   at System.Web.Http.Owin.OwinHttpRequestContext.get_IncludeErrorDetail()
   at System.Net.Http.HttpRequestMessageExtensions.ShouldIncludeErrorDetail(HttpRequestMessage request)
   at System.Net.Http.HttpRequestMessageExtensions.CreateErrorResponse(HttpRequestMessage request, HttpStatusCode statusCode, Func`2 errorCreator)
   at System.Net.Http.HttpRequestMessageExtensions.CreateErrorResponse(HttpRequestMessage request, HttpStatusCode statusCode, Exception exception)
   at System.Web.Http.ExceptionHandling.DefaultExceptionHandler.Handle(ExceptionHandlerContext context)
   at System.Web.Http.ExceptionHandling.DefaultExceptionHandler.HandleAsync(ExceptionHandlerContext context, CancellationToken cancellationToken)
   at System.Web.Http.ExceptionHandling.LastChanceExceptionHandler.HandleAsync(ExceptionHandlerContext context, CancellationToken cancellationToken)
   at System.Web.Http.ExceptionHandling.ExceptionHandlerExtensions.<HandleAsyncCore>d__0.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Web.Http.Dispatcher.HttpControllerDispatcher.<SendAsync>d__1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Web.Http.HttpServer.<SendAsync>d__0.MoveNext()
2017-04-28 14:49:54.8083|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|Exception during request processing.
System.ObjectDisposedException: Cannot access a disposed object.
Object name: 'System.Net.HttpListenerRequest'.
   at System.Net.HttpListenerRequest.CheckDisposed()
   at System.Net.HttpListenerRequest.get_LocalEndPoint()
   at System.Net.HttpListenerRequest.get_IsLocal()
   at Microsoft.Owin.Host.HttpListener.RequestProcessing.OwinHttpListenerContext.GetServerIsLocal()
   at Microsoft.Owin.Host.HttpListener.RequestProcessing.CallEnvironment.get_ServerIsLocal()
   at Microsoft.Owin.Host.HttpListener.RequestProcessing.CallEnvironment.PropertiesTryGetValue(String key, Object& value)
   at Microsoft.Owin.Host.HttpListener.RequestProcessing.CallEnvironment.TryGetValue(String key, Object& value)
   at Microsoft.Owin.OwinContext.Get[T](String key)
   at System.Web.Http.Owin.OwinHttpRequestContext.get_IsLocal()
   at System.Web.Http.Owin.OwinHttpRequestContext.get_IncludeErrorDetail()
   at System.Web.Http.ExceptionHandling.LastChanceExceptionHandler.CreateDefaultLastChanceResult(ExceptionContext context)
   at System.Web.Http.ExceptionHandling.LastChanceExceptionHandler.HandleAsync(ExceptionHandlerContext context, CancellationToken cancellationToken)
   at System.Web.Http.ExceptionHandling.ExceptionHandlerExtensions.<HandleAsyncCore>d__0.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Web.Http.HttpServer.<SendAsync>d__0.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Web.Http.Owin.HttpMessageHandlerAdapter.<InvokeCore>d__0.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Owin.AutofacAppBuilderExtensions.<>c__DisplayClass8_0.<<RegisterAutofacLifetimeScopeInjector>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.Owin.Host.HttpListener.OwinHttpListener.<ProcessRequestAsync>d__5.MoveNext()
2017-04-28 15:40:05.2985|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|Exception during request processing.
System.AggregateException: One or more errors occurred. ---> System.Threading.Tasks.TaskCanceledException: A task was canceled.
   --- End of inner exception stack trace ---
   at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   at System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   at Microsoft.Owin.StaticFiles.SendFileMiddleware.SendFileWrapper.<>c__DisplayClass2.<SendAsync>b__0(Task resultTask)
   at System.Threading.Tasks.ContinuationTaskFromTask.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.Owin.Host.HttpListener.OwinHttpListener.<ProcessRequestAsync>d__5.MoveNext()
---> (Inner Exception #0) System.Threading.Tasks.TaskCanceledException: A task was canceled.<---

2017-04-28 15:40:05.2985|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|Exception during request processing.
System.OperationCanceledException: The operation was canceled.
   at System.Threading.CancellationToken.ThrowOperationCanceledException()
   at System.Web.Http.Owin.HttpMessageHandlerAdapter.<BufferResponseContentAsync>d__13.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Web.Http.Owin.HttpMessageHandlerAdapter.<InvokeCore>d__0.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Owin.AutofacAppBuilderExtensions.<>c__DisplayClass8_0.<<RegisterAutofacLifetimeScopeInjector>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.Owin.Host.HttpListener.OwinHttpListener.<ProcessRequestAsync>d__5.MoveNext()
2017-04-28 15:45:38.1964|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-28 15:45:43.1978|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-28 15:45:43.1978|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-28 15:45:43.1978|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-28 15:45:43.2028|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-28 15:45:48.1989|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-28 16:52:38.6135|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-28 16:52:38.6376|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-28 18:30:19.3241|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
2017-04-28 18:30:19.3569|ERROR|Microsoft.Owin.Host.HttpListener.OwinHttpListener|IOCompletionCallback
System.ComponentModel.Win32Exception (0x80004005): The specified network name is no longer available
